src/realm/object-store/object_store.cpp

```diff
@@ -959,25 +939,6 @@ Schema ObjectStore::schema_from_group(Group const& group)
     return schema;
 }
 
-util::Optional<Property> ObjectStore::property_for_column_key(ConstTableRef& table, ColKey column_key)
```

Comment:
This was added for Java during the core 6 port but ended up being unneeded and has never been used.